server/gokbg3/grails-app/controllers/org/gokb/IntegrationController.groovy

```diff
@@ -1559,13 +1568,13 @@ class IntegrationController {
                       'reasonRetired'
                 ], titleObj, title)
 
-                if (titleObj.type == 'Serial') {
-                  def pubFrom = GOKbTextUtils.completeDateString(titleObj.publishedFrom)
-                  def pubTo = GOKbTextUtils.completeDateString(titleObj.publishedTo, false)
+                def pubFrom = GOKbTextUtils.completeDateString(titleObj.publishedFrom)
+                def pubTo = GOKbTextUtils.completeDateString(titleObj.publishedTo, false)
 
-                  title_changed |= ClassUtils.setDateIfPresent(pubFrom, title, 'publishedFrom')
-                  title_changed |= ClassUtils.setDateIfPresent(pubTo, title, 'publishedTo')
-                }
+                log.debug("Completed date publishedFrom ${titleObj.publishedFrom} -> ${pubFrom}")
+
+                title_changed |= ClassUtils.setDateIfPresent(pubFrom, title, 'publishedFrom')
+                title_changed |= ClassUtils.setDateIfPresent(pubTo, title, 'publishedTo')
```

Comment:
Like the date props the new title String properties (`series` and `subjectArea`) should be updated around here as well. Instead of setDateIfPresent for that we would use

https://github.com/openlibraryenvironment/gokb/blob/0a2a234c16b61a5501bbcfda722c26ed67cef655/server/gokbg3/src/main/groovy/com/k_int/ClassUtils.groovy#L174